src/main/scala/chiseltest/internal/Testers2.scala

```diff
@@ -41,6 +41,16 @@ case object CachingAnnotation extends TestOptionObject {
   )
 }
 
+case object UseThreads extends TestOptionObject {
```

Comment:
Here is the `UseThreads` annotation that I was talking about at the dev meeting today.